api/src/main/java/org/apache/iceberg/expressions/ManifestEvaluator.java

```diff
@@ -250,11 +251,33 @@ public Boolean or(Boolean leftResult, Boolean rightResult) {
 
     @Override
     public <T> Boolean in(BoundReference<T> ref, Set<T> literalSet) {
+      int pos = Accessors.toPosition(ref.accessor());
+      PartitionFieldSummary fieldStats = stats.get(pos);
+      if (fieldStats.lowerBound() == null) {
+        return ROWS_CANNOT_MATCH; // values are all null and literal cannot contain null
+      }
```

Comment:
There should be a newline after control flow statements like this one.

Reply:
Done.